charmhelpers/contrib/openstack/context.py

```diff
@@ -521,6 +526,64 @@ def __call__(self):
         return {}
 
 
+class NovaVendorMetadataContext(OSContextGenerator):
```

Comment:
I'm a bit confused as to why this is called the `NovaVendor...` rather than just `VendorMeta...` context.  There doesn't seem to be anything specific in the class that is referring to nova?  If it is more generalised, then adding `Nova` may confuse things in the future?

Reply:
Alex, I thought about naming NovaVendorMetadata so it is easier to know this would apply to Nova Metadata API service, and the context returned is aimed at nova.conf file.

Reply:
Rodrigo, ah, that is a good reason.  Let's keep it as is.